app/(gcforms)/[locale]/(form administration)/form-builder/[id]/edit/components/elements/element-dialog/ElementDialog.tsx

```diff
@@ -8,18 +8,27 @@ import { Groups } from "@lib/hooks/form-builder/useElementOptions";
 import { ElementFilters } from "./ElementFilters";
 import { Dialog, useDialogRef } from "@formBuilder/components/shared/Dialog";
 import { ListBox } from "@formBuilder/components/shared/ListBox";
+import { useFeatureFlags } from "@lib/hooks/useFeatureFlags";
 
 export type SelectedGroupState = {
   group: Groups | "all";
   ref: React.RefObject<HTMLElement>;
 };
 
-const filterElementsByGroup = (elements: ElementOption[], selectedGroup: Groups | "all") => {
+const filterElementsByGroup = (
+  elements: ElementOption[],
+  selectedGroup: Groups | "all",
+  featureFlags: { [key: string]: boolean }
+) => {
+  const excludeId = featureFlags["addressComplete"] ? "address" : "addressComplete";
```

Comment:
Maybe rename this --- to excludeAddressComplete vs excludeId to make the filter a little more clear.